src/components/Lookup/styles.scss

```diff
@@ -317,3 +317,180 @@
         fill: $color-gray-4;
     }
 }
+
+.rainbow-lookup_selected-value--readonly {
+    display: flex;
+
+    .rainbow-lookup_selected-value-icon {
+        fill: #a4a7b5;
+        height: 100%;
+        width: 22px;
+        top: 0;
+        position: absolute;
+        line-height: 1;
+        border: 0;
+        z-index: 2;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+
+        svg {
+            width: 16px;
+            height: 16px;
+            font-size: 16px;
+        }
+    }
+
+    .rainbow-lookup_selected-value-icon:not(button) {
+        pointer-events: none;
+    }
+
+    .rainbow-lookup_selected-value-input {
+        box-shadow: none;
+        caret-color: transparent;
+        background-color: transparent;
+        border: 1px solid transparent;
+        font-size: $font-size-text-large;
+        font-weight: 400;
+        display: inline-block;
+
+        &[type='text']::-ms-clear {
+            display: none;
+            width: 0;
+            height: 0;
+        }
+
+        &:focus,
+        &:active {
+            outline: 0;
+            border: 1px solid transparent;
+            box-shadow: none;
+            border-color: transparent;
+            background-color: transparent;
+        }
+
+        &::placeholder {
+            color: $color-gray-3;
+        }
+
+        &[readonly] {
+            outline: 0;
+            background-color: transparent;
+            border: 2px solid transparent;
+            color: $color-dark-1;
+            cursor: not-allowed;
+            -webkit-user-select: none;
+            -moz-user-select: none;
+            -ms-user-select: none;
+            user-select: none;
+            padding-left: 0;
+            padding-right: 0;
+
+            &:focus,
+            &:active {
+                outline: 0;
+                box-shadow: none;
+                border: 2px transparent solid;
+            }
+        }
+    }
+
+    .rainbow-lookup_selected-value-input-with-icon {
+        &[readonly] {
+            padding-left: 1.75rem;
+
+            &:focus,
+            &:active {
+                padding-left: 1.75rem;
+            }
+        }
+    }
+}
+
+.rainbow-lookup_selected-value--disabled {
```

Comment:
I think that this className is not necessary, you only need use the `[disabled]` selector in the `rainbow-lookup_selected-value` className